functions/go/inflate-helm-chart/README.md

```diff
@@ -0,0 +1,237 @@
+`configHome`   | (exec only) Defines a value that the function should pass to helm via the HELM_CONFIG_HOME environment variable. If omitted, {tmpDir}/helm is used, where {tmpDir} is some temporary directory created by the function for the benefit of helm.
+`name`         | The name of the chart
+`version`      | The version of the chart
+`repo`         | A URL locating the chart on the internet
+`releaseName`  | Replaces RELEASE_NAME in the chart template output
+`namespace`    | Sets the target namespace for a release (.Release.Namespace in the template)
+`valuesInline` | Values to use instead of default values that accompany the chart
+`valuesFile`   | valuesFile is a remote or local file path to a values file to use _instead of_ the default values that accompanied the chart. The default values are in '{ChartHome}/{Name}/values.yaml'.
+`valuesMerge`  | ValuesMerge specifies how to treat ValuesInline with respect to Values. Legal values: 'merge', 'override' (default), 'replace'.
+`includeCRDs`  | Specifies if Helm should also generate CustomResourceDefinitions, defaults to 'false'
+
+The only required field is `name`.
+
+<!--mdtogo-->
+
+## Examples
+
+<!--mdtogo:Examples-->
+
+### Example with kpt
+
+To inflate a remote minecraft chart, you can run the following command: 
+
+```shell
+$ kpt fn eval --image gcr.io/kpt-fn/inflate-helm-chart:unstable --network -- \
+name=minecraft \
+repo=https://itzg.github.io/minecraft-server-charts \
+releaseName=test
+```
+
+This will add two files to your directory, which you can view:
+
+```shell
+$ kpt pkg tree
+├── [secret_test-minecraft.yaml]  Secret test-minecraft
+└── [service_test-minecraft.yaml]  Service test-minecraft
+```
+
+### Example with kustomize
+You can specify your `functionConfig` via the `generators` field in your `kustomization.yaml` file:
+
+The `functionConfig` can be of type `InflateHelmChart`:
+
+```yaml
+generators:
+- |-
+  apiVersion: v1
+  kind: InflateHelmChart
+  metadata:
+    name: demo
+    annotations:
+      config.kubernetes.io/function: |
+        container:
+          network: true
+          image: gcr.io/kpt-fn/inflate-helm-chart:latest
+  helmCharts:
+  - name: minecraft
+    repo: https://itzg.github.io/minecraft-server-charts
+    version: 3.1.3
+    releaseName: test
+```
+
+You can equivalently use a `functionConfig` of type `ConfigMap`:
+
+```yaml
+generators:
+- |-
+  apiVersion: v1
+  kind: ConfigMap
+  metadata:
+    name: demo
+    annotations:
+      config.kubernetes.io/function: |
+        container:
+          network: true
+          image: gcr.io/kpt-fn/inflate-helm-chart:unstable
+  data:
+    name: minecraft
+    repo: https://itzg.github.io/minecraft-server-charts
+    version: 3.1.3
+    releaseName: test
+```
+
+For both of the above kustomizations, the helm chart can be
+inflated with the following command:
+
+```shell
+kustomize build --enable-alpha-plugins --network .
```

Comment:
If a minimum kustomize is required, please specify here.

Reply:
Added kustomize v4 to specify